crates/turbopack-ecmascript/src/chunk/optimize.rs

```diff
@@ -1,4 +1,7 @@
-use std::{cmp::Ordering, collections::HashSet};
+use std::{
```

Comment:
would be nice to add some module level docs that give a high level overview of what's happening here (same as above)